src/frequenz/sdk/timeseries/_formula_engine/_formula_steps.py

```diff
@@ -132,6 +132,52 @@ def apply(self, eval_stack: List[float]) -> None:
         eval_stack.append(res)
 
 
+class Maximizer(FormulaStep):
+    """A formula step that represents the max function."""
+
+    def __repr__(self) -> str:
+        """Return a string representation of the step.
+
+        Returns:
+            A string representation of the step.
+        """
+        return "max"
```

Comment:
Not related to this PR, but do you know why all `FormulaStep` are using `__repr__` rather than `__str__`? It seems to me `__str__` is more suitable for these use-cases but I might be missing something.

Reply:
You are right. The way we use it, it seems that `__str__`is mor suitable. I will not touch it in this PR though and rather change it later.

Reply:
Actually that is used when pushing operators to the stack and there is a dict with corresponding names. So it makes sense after all.

Reply:
we can still rename them to longer names representing the classes and add str representation. I've created an issue for later :).
https://github.com/frequenz-floss/frequenz-sdk-python/issues/614